opensrp-chw/src/at/assets/json.form/family_details_remove_member_sw.json

```diff
@@ -93,10 +93,10 @@
         "openmrs_entity_id": "160417AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
         "openmrs_data_type": "select one",
         "type": "spinner",
-        "hint": "Sababu ya kuiondoa",
+        "hint": "Sababu ya Kumuondoa",
         "v_required": {
           "value": "true",
-          "err": "Chagua sababu ya kuondoa rekodi ya familia"
+          "err": "Chagua sababu ya kumuondoa rekodi ya familia"
```

Comment:
Chagua sababu ya kumuondoa 'kwenye' rekodi ya familia may sounds a lot better I don't know what the program team thinks.

Reply:
Perfect, you decide. Comments related  were here https://docs.google.com/spreadsheets/d/1VCon-DUaiw4KKo4rD1mTFtzH4fAvnT-91lhECuS-o4Y/edit#gid=524880454&range=D99  but I see it's not exactly this string. Also check that length does not break because I'm not sure where this is shown.